process/factory/shard/vmContainerFactory.go

```diff
@@ -5,12 +5,8 @@ import (
 	"io"
 	"sort"
 
-	arwen12 "github.com/ElrondNetwork/arwen-wasm-vm/v1_2/arwen"
-	arwenHost12 "github.com/ElrondNetwork/arwen-wasm-vm/v1_2/arwen/host"
-	arwen13 "github.com/ElrondNetwork/arwen-wasm-vm/v1_3/arwen"
-	arwenHost13 "github.com/ElrondNetwork/arwen-wasm-vm/v1_3/arwen/host"
-	arwen14 "github.com/ElrondNetwork/arwen-wasm-vm/v1_4/arwen"
-	arwenHost14 "github.com/ElrondNetwork/arwen-wasm-vm/v1_4/arwen/host"
+	arwen14 "github.com/ElrondNetwork/wasm-vm/arwen"
+	arwenHost14 "github.com/ElrondNetwork/wasm-vm/arwen/host"
```

Comment:
goimports

Reply:
Fixing on a separate branch, for all files in the repo.